_posts/developers/apps/vocabularies/2019-01-01-02_quickstart-usage.md

```diff
@@ -1,10 +1,12 @@
 ---
 layout: for-developers
 title: "Describe data"
-permalink: /for-developers/apps/vocabularies/use/quickstart
+permalink: /developers/vocabularies/use/quickstart
 tags: [apps]
 categories: [Vocabularies, Quickstart]
 exclude: true
+redirect_from:
+  - /developers/vocabularies/use/quickstart
```

Comment:
Shouldn't the `redirect_from` be from the previous link ?

Reply:
That is correct.

Reply:
Yeah I'm realizing that I did a global search and replace at one point and ended up changing these accidentally.